README.md

```diff
@@ -1,30 +1,45 @@
-## Welcome to the TomoPy benchmarking repository
+# Welcome to the TomoPy benchmarking repository
 
-### Introduction
+## Introduction
 
 This website contains all of the benchmarking data for the latest stable version of TomoPy. If you are interested in older versions, go to the [tomopy.github.io](https://github.com/tomopy/tomopy.github.io) repository. 
 
 These benchmarks are designed to give the reader the information they need to make an informed decision about which of TomoPy's reconstruction algorithms are right for their particular project. To accomplish that, these benchmarks contain results pertaining to the quality and speed of each reconstruction algorithm as well as the types of artifacts that may occur when using each of the algorithms. 
 
-### Data generation
+## Results
+To see how the iterative reconstruction algorithms improve with each iteration, you see the [summary graph](https://github.com/tomopy/tomopy.github.io/blob/main/2021-02-19/cpu/peppers/summary.svg) after each benchmark. To quantitatively measure quality, we use the multi-scale structural similarity index (MS-SIM) because it has been shown to have a stronger correlation to human perceptions of quality than simpler quality measures like the mean-squared error. 
```

Comment:
Needs more explanation about how to interpret the benchmark. Is a MS-SSIM of 1 good or bad? Is 1 attainable? "SSIM" has two "S". MS-SSIM is a full-reference metric; you need to explain what images are being rated as 'similar'.